editor/src/core/es-modules/package-manager/package-manager.spec.ts

```diff
@@ -134,13 +122,14 @@ describe('ES Dependency Manager — Real-life packages', () => {
       fail(`Expected successful nodeModules fetch`)
     }
     const nodeModules = fetchNodeModulesResult.nodeModules
-    const req = getRequireFn(moduleUpdater(nodeModules), nodeModules)
+    const onRemoteModuleDownload = jest.fn()
```

Comment:
you don't seem to be passing `onRemoteModuleDownload` anywhere, so the `expect(onRemoteModuleDownload).toBeCalledTimes(0)` is not really testing anything

Reply:
Ooops

Reply:
Fixed